### PR TITLE
Update for Martha release notes [BT-167]

### PR DESCRIPTION
**Pull request by grsterin**

### CODE REVIEW

**Comment by grsterin** [comment]
jenkins retest

File: docs/release-checklist.md [hunk 1]
```diff
@@ -8,27 +8,28 @@ Martha Release Checklist
...
-        - You will need to be on the Broad network in order to have access.
-    1. [Run the full manual test on the dev environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)
-1. Deploy and test on the other non-prod environments
-    1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "staging" as the TARGET.  Do not proceed until the job ends successfully.
-    1. Link your test account to your ERA Commons account on staging (see [the link your account directions within the manual test doc](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)) and run the following commands in a terminal
+- [ ] Create a Github ticket for Martha release and copy this checklist there.
```
Comment (mcovarr) [inline docs/release-checklist.md]: Just to confirm, this really means Github and not JIRA? Edit: probably so if it's going to be a checklist, but I thought we left Github behind for issue tracking...

File: docs/release-checklist.md [hunk 2]
```diff
@@ -8,27 +8,28 @@ Martha Release Checklist
...
+- [ ] Deploy and test on the other non-prod environments  
+    1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "staging" as the TARGET.  Do not proceed until the job ends successfully.  
+    1. Link your test account to your ERA Commons account on staging (see [the link your account directions within the manual test doc](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)) and run the following commands in a terminal  
          ```
          gcloud auth login <your test account email>
          curl -X POST https://us-central1-broad-dsde-staging.cloudfunctions.net/martha_v3 -H "Authorization: Bearer $(gcloud auth print-access-token)" -d '{"url": "drs://dg.712C/fa640b0e-9779-452f-99a6-16d833d15bd0"}' -H "Content-Type: application/json" | jq
          ```
-    1. You should get a JSON object in response with the following structure:
+    1. You should get a JSON object in response with the following structure:
```
Comment (mcovarr) [inline docs/release-checklist.md]: revert

File: docs/release-checklist.md [hunk 3]
```diff
@@ -38,13 +39,13 @@ Martha Release Checklist
           }
         }
         ```
-    1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "alpha" as the TARGET.  Do not proceed until the job ends successfully.
-    1. Link your test account to your ERA Commons account on alpha (see [the link your account directions within the manual test doc](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)) and run the following commands in a terminal
+    1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "alpha" as the TARGET.  Do not proceed until the job ends successfully.
```
Comment (mcovarr) [inline docs/release-checklist.md]: revert this and the line below

File: docs/release-checklist.md [hunk 4]
```diff
@@ -38,13 +39,13 @@ Martha Release Checklist
...
+    1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "alpha" as the TARGET.  Do not proceed until the job ends successfully.  
+    1. Link your test account to your ERA Commons account on alpha (see [the link your account directions within the manual test doc](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)) and run the following commands in a terminal  
          ```
          gcloud auth login <your test account email>
          curl -X POST https://us-central1-broad-dsde-alpha.cloudfunctions.net/martha_v3 -H "Authorization: Bearer $(gcloud auth print-access-token)" -d '{"url": "drs://dg.712C/fa640b0e-9779-452f-99a6-16d833d15bd0"}' -H "Content-Type: application/json" | jq
          ```
-    1. You should get a JSON object in response with the following structure:
+    1. You should get a JSON object in response with the following structure:
```
Comment (mcovarr) [inline docs/release-checklist.md]: revert

File: docs/release-checklist.md [hunk 5]
```diff
@@ -54,7 +55,7 @@ Martha Release Checklist
           }
         }
         ```
-1. Deploy and test on prod (including manual test)
-     1. Go to the [Jenkins prod manual deploy project](https://fcprod-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created (you shouldn't have a choice for TARGET).  Do not proceed until the job ends successfully.
-         - You will need to be on the Broad network in order to have access.
-     1. [Run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)
+- [ ] Deploy and test on prod (including manual test)
```
Comment (mcovarr) [inline docs/release-checklist.md]: remove trailing whitespace in these changes

**Review by mcovarr — APPROVED** [review]

**Comment by kshakir** [comment]
jenkins retest as something somewhere reverted back to returning 500s again. yay?

**Review by kshakir — COMMENTED** [review]
I'm happy when Gary gives his 👍

File: docs/release-checklist.md [hunk 6]
```diff
@@ -8,20 +8,21 @@ Martha Release Checklist
...
     - e.g.: `gcloud container images describe us.gcr.io/broad-dsp-gcr-public/martha:1.5.2`
-1. Deploy and test on dev (including manual test)
+- [ ] Deploy and test on dev (including manual test)
     1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "dev" as the TARGET.  Do not proceed until the job ends successfully.
         - You will need to be on the Broad network in order to have access.
     1. [Run the full manual test on the dev environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)
-1. Deploy and test on the other non-prod environments
+- [ ] Deploy and test on the other non-prod environments
     1. Go to the [Jenkins manual deploy project](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created and "staging" as the TARGET.  Do not proceed until the job ends successfully.
```
Comment (kshakir) [inline docs/release-checklist.md]: ToL: While you're here, via the [Release Testing Guidelines](https://docs.google.com/document/d/1Ej50XUEPwLQtjK6mQF9-xx7DuXyIhEs3UP8aii0bpsk/edit) one should be releasing to alpha _before_ staging, not after.

But that can be a separate ticket.

release environments

Comment (grsterin) [reply docs/release-checklist.md]: I've made an update in this PR

**Review by gdlex4015 — APPROVED** [review]

**Pull request merged**